fidimag/atomistic/atomistic_driver.py

```diff
@@ -133,7 +133,11 @@ def set_default_options(self, gamma=1, mu_s=1, alpha=0.1):
         # When we create the simulation, mu_s is set to the default value. This
         # is overriden when calling the set_mu_s method from the Simulation
         # class or when setting mu_s directly (property)
+        # David Tue 19 Jun 2018: Not a very clear thing to do, we must set a
+        # WARNING
         self._mu_s[:] = mu_s
+        self._mu_s_inv[:] = 1. / mu_s
```

Comment:
I don't think this is sufficient - if on a given site mu_s = 0, this will lead to weird behaviour.

Reply:
That line `139` shouldn't be there, since `mu_s_inv` is created in `common/sim_base`

Reply:
Ok, but surely l 138 should also be removed then?

Reply:
I guess so, at least we are inheriting from base or another class? There should be references to the magnetisation arrayys

Reply:
Mmm yes this needs to change then so it is a definitely a reference.

I'm going to write some tests now to check that it's a reference in the driver/energy classes so we make sure we don't have this problem again.